src/main/resources/text/helpForAllCommands.txt

```diff
@@ -0,0 +1,41 @@
+
+Contact commands
+----------------------
+contact add	                Adds a contact                                              contact add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]… [r/REMARK]…
+contact edit                Edits an existing contact                                   contact edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]… [r/REMARK]…
+contact list                Lists all contacts                                          contact list
+contact find                Find contacts given search keyword(s)                       contact find KEYWORD [MORE_KEYWORDS]
+contact delete              Deletes a contact                                           contact delete INDEX
+
+Tag commands
+----------------------
+tag add	                    Adds a tag 	                                                tag add t/TAG
```

Comment:
Can remove the tag add command since we will not be supporting it.

Reply:
I will be creating PRs to update this file before every end of milestone!